Dockerfile

```diff
@@ -59,9 +59,9 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-ins
     python-is-python3 \
     git \
     pkg-config \
+    ninja-build \
     libboost-dev \
     libboost-program-options-dev \
```

Comment:
Are you able to remove this now that you're passing `-Dtools=disabled`?

Reply:
If I remove that line, I stiill get the "ERROR: Could not detect Ninja v1.8.2 or newer" error.

Reply:
I think you're looking at the wrong line. I'm talking about libboost-program-options-dev.

Reply:
Ah. Let me give that a try quickly.

Reply:
The fsim still seems to need it.